docs/readmes/dp/debug_logs.md

```diff
@@ -7,7 +7,111 @@ id: debug_logs title: View Logs hide_title: true
+    1. `2022-04-25 15:41:22,664 DEBUG Sending TR069 message: GetParameterValues {...}`
+    1. `2022-04-25 15:41:22,671 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 3242`
+    1. `2022-04-25 15:41:22,718 DEBUG Handling TR069 message: GetParameterValuesResponse {...}`
+    1. `2022-04-25 15:41:22,719 DEBUG Received CPE parameter values: {...}`
+    1. `2022-04-25 15:41:22,719 DEBUG State transition from <WaitGetParametersState> to <get_obj_params>`
+    1. `2022-04-25 15:41:22,720 DEBUG State transition from <GetObjectParametersState> to <wait_get_obj_params>`
+    1. `2022-04-25 15:41:22,720 DEBUG Sending TR069 message: GetParameterValues {...}`
+    1. `2022-04-25 15:41:22,723 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 877`
+    1. `2022-04-25 15:41:22,769 DEBUG Handling TR069 message: GetParameterValuesResponse {...}`
+    1. `2022-04-25 15:41:22,770 DEBUG Received object parameters: {...}`
+
+1. `SetParameterValues` message followed by `GetParameterValues` - conditional, only appears if a configuration change is needed to be done on the eNB side
+(the actual parameters that are set in order to enable transmission based on Domain Proxy grant may differ depending on the eNB model and eNB current configuration state - please refer to your eNB device model implementation
+in `lte/gateway/python/magma/enodebd/devices`)
+    1. `2022-04-25 15:41:22,771 DEBUG State transition from <WaitGetObjectParametersState> to <set_params>`
+    1. `2022-04-25 15:41:22,772 DEBUG Sending TR069 request to set CPE parameter values: {...}`
+    1. `2022-04-25 15:41:22,773 DEBUG State transition from <SetParameterValuesState> to <wait_set_params>`
+    1. `2022-04-25 15:41:22,773 DEBUG Sending TR069 message: SetParameterValues {...}`
+    1. `2022-04-25 15:41:22,777 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 723`
+    1. `2022-04-25 15:41:22,837 DEBUG Handling TR069 message: SetParameterValuesResponse {'Status': '0'}`
+    1. `2022-04-25 15:41:22,838 INFO Successfully configured CPE parameters!`
+    1. `2022-04-25 15:41:22,839 DEBUG State transition from <WaitSetParameterValuesState> to <check_get_params>`
+    1. `2022-04-25 15:41:22,840 DEBUG State transition from <GetParametersState> to <check_wait_get_params>`
+    1. `2022-04-25 15:41:22,840 DEBUG Sending TR069 message: GetParameterValues {...}`
+    1. `2022-04-25 15:41:22,849 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 3242`
+    1. `2022-04-25 15:41:22,916 DEBUG Handling TR069 message: GetParameterValuesResponse {...}`
+1. Update transmission configuration from Domain Proxy and TR069 session end - `notify_dp` is a `enodebd` transition state that calls Domain Proxy `GetCBSDState` gRPC API. `GetCBSDState` response contains data, which indicate whether eNB radio should disabled/turned of or enabled together with transmission parameters like frequency, bandwidth, power. This configuration will be applied to eNB on next tr069 session in `SetParameterValues` step
```

Comment:
Mind the sentences, please. It is hard to understand otherwise.
- Full stop instead of `-`.
- `turned off`
- In a sentence `and` should be used before the last item in a list.
- What does `this configuration` refer to. It is ambivalent. "call the devil by its name", please.
- `step.` Full stop.